api/api.go

```diff
@@ -128,11 +129,25 @@ func parseAsUintValue(sv string, min, max, def uint) uint {
 		return max
 	}
 	if min != 0 && uint(iv) < min {
-		return max
+		return min
 	}
 	return uint(iv)
 }
 
+func parseAsIntValue(sv string, min, max, def int) int {
```

Comment:
Could I trouble you to write a few tests on the bounds checking logic for this and for `parseAsUintValue`?